.github/workflows/tests.yml

```diff
@@ -2,15 +2,51 @@ name: tests
 
 on: [ pull_request ]
 
+
+env:
+  PKG_NAME: peerdid
+
+
 jobs:
-  build:
 
+  check-version-bumped:
+    name: Check version bumped
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.event.pull_request.base.ref == 'main'
```

Comment:
I think we need to check version only if we send a PR to `stable`, don't we? We don't need to bump the version at any PR to `main`.

Reply:
yes, I missed that place. Thanks